README.md

```diff
@@ -16,13 +16,23 @@ DefTest is provided as a Defold library project for easy integration in your own
 	https://github.com/britzl/deftest/archive/master.zip
 
 ```Lua
-It is recommended to run your unit tests from its own collection, set as the bootstrap collection in game.project. Add a game object and a script to the collection and use the script to set up your tests. An example:
+It is recommended to run your unit tests from its own collection, set as the bootstrap collection in game.project. Add a game object and a script to the collection and use the script to set up your tests. Add objects, which scripts you want to test.
+
+For running integration tests, it is important to add integration_test_manager.go located in the /deftest/integration/ folder to the collection and link it to deftest with deftest.set_integration_test_manager(path_to_integration_manager_object).
+
+deftest.add() adds unit tests, which test pure logic.
+deftest.add_integration() adds tests, which can be used to test scripts.
+
+An example:
 
 	local deftest = require "deftest.deftest"
 	local some_tests = require "test.some_tests"
 	local other_tests = require "test.other_tests"
+	local integration_tests = require "test.some_integration_tests"
 
 	function init(self)
+		deftest.set_integration_test_manager("/integration_test_manager#integration_test_manager")
```

Comment:
Could this perhaps be done automatically from the integration_test_manager.script in init()?

Reply:
That's a good point!